index.bs

```diff
@@ -11290,6 +11290,35 @@ defined below, skipping over those channels not present.
 		<tr><td>17 <td>SPEAKER_TOP_BACK_RIGHT <td> <td> <td> <td>
 </table>
 
+<h3 id="channels-tail-time">
+Implication of tail-time on input and output channel count</h3>
+
+When an {{AudioNode}} has a non-null <a>tail-time</a>, and an output channel
+count that depends on the input channels count, any change in input channel
+count MUST take into account the {{AudioNode}}'s <a>tail-time</a>.
```

Comment:
Perhaps:

`...MUST take the {{AudioNode}}'s <a>tail-time</a> into account.`

Reply:
Better indeed, thanks.